sopel/irc/backends.py

```diff
@@ -57,7 +57,7 @@ def __init__(
     ):
         super().__init__(bot)
 
-    def is_connected(self) -> False:
+    def is_connected(self) -> bool:
```

Comment:
Oops, this was my fault from #2394, wasn't it. x)

Reply:
It happens.